modules/serverless-deleting-knative-serving-crds.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="deleting-knative-serving-crds_{context}"]
+= Deleting Knative Serving CRDs from the Operator
+
+After uninstalling the {ServerlessOperatorName}, the Operator CRDs and API services remain on the cluster. Use this procedure to completely uninstall the remaining components. 
+
+.Prerequisite
+*  You have uninstalled Knative Serving and removed the {ServerlessOperatorName} using the procedure above. 
```

Comment:
Per the IBM style guide, don't use `above`. Use `previously` or `preceding` to indicate position.